tests/InputFiles/ExampleExecutables/RandomAmr1D.yaml

```diff
@@ -3,10 +3,11 @@
 
 Executable: RandomAmr1D
 Testing:
-  Check: parse;execute_check_output
+  Check: parse;execute
+  Timeout: 10
+  ExpectedExitCode: 2
```

Comment:
why is it 2?  How would I determine this?

Reply:
Added docs to `CheckpointAndExitAfterWallclock`